### PR TITLE
Toggleable comments & Question deletion

### PR DESCRIPTION
**Pull request by TGRHavoc**
I've added toggleable comments to /views & clarified a few things in the README 

Added a delete button to /editor so admins can delete questions without
submitting the form with empty question. (See #6)

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-listen_port = 8214
+listen_port = 8212
```
Comment (marc-hanheide) [inline config.py:1]: May I ask why this change is included?

**Comment by marc-hanheide** [comment]
Nice work. Thanks for this. I shall give it a try before merging, but a quick look through looks great.

**Comment by marc-hanheide** [comment]
And... good to see such an active GitHub profile of one of our students :+1:

File: config.py (hunk 1 above)
Comment (TGRHavoc) [inline config.py:1]: It's only included to keep the default values in line with the example you've given in the readme, I feel like it better clarifies what the variable does (You can remove it if you want :) )

**Comment by TGRHavoc** [comment]
> Nice work. Thanks for this. I shall give it a try before merging, but a quick look through looks great

Thanks :grinning: glad I could contribute in some way.

**Comment by marc-hanheide** [comment]
works perfectly, I'll merge.

**Comment by marc-hanheide** [comment]
deployed to https://lcas.lincoln.ac.uk/qv/ Thanks again.
